bin/webpack.js

```diff
@@ -180,7 +188,7 @@ yargs.parse(process.argv.slice(2), (err, argv, output) => {
 		}
 	}
 
-	function processOptions(options) {
+	function processOptions(options, insightReporter) {
```

Comment:
`insight`=> `?node >= v6.0.0` => `function processOptions(options, { reporter, events }) // param destruct`

Reply:
e.g 

```js
reporter.trackEvent(
  events.onConfig("options-collected", {
    label: JSON.stringify(options),
    value: 1
  })
);
```

Reply:
I'm wanting this to land in v3 so still v4 support at least.

Reply:
Yup, I expressed my intend in a weirdly manner, all I meant is instead of 

```
js
// from 
const Insight = ...
const insightEvents = ...

// to
const insight = { ctor, events }

// node =< 4
// function (processOptions, insight) (pass {Object})
insight.reporter.trackEvent(
  insight.events.onConfig("options-collected", {
    label: JSON.stringify(options),
    value: 1
  })
);

// node =< 6  
// function (processOptions, { reporter, events }) (pass {Object} with destruct)
reporter.trackEvent(
  events.onConfig("options-collected", {
    label: JSON.stringify(options),
    value: 1
  })
);
``` 

Just nitpicking 🐦 ... 😛